### ISSUE TITLE
[Bug]: An error: expected unhandled field is reported when inserting Array rows data

### ISSUE THREAD

**Issue by ThreadDao** [milvus-io/milvus-sdk-go#609]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

- milvus image: v2.3.2
- go-sdk branch: master

Test steps:
1. Create a collection with some fields of type Array
2. Insert array rows data gets error: expected unhandled field, It seems that the go-SDK does not implement row insertion of type Array.
In the /milvus-sdk-go/entity/rows.go file, the FieldTypeArray type case is not implemented.
![image](https://github.com/milvus-io/milvus-sdk-go/assets/27288593/24be25f8-2915-4eec-af45-1f90baf393d1)

Test cases:
```
func TestInsertArrayRows(t *testing.T) {
	ctx := createContext(t, time.Second*common.DefaultTimeout)
	// connect
	mc := createMilvusClient(ctx, t)

	// create collection
	cp := CollectionParams{CollectionFieldsType: AllFields, AutoID: false, EnableDynamicField: false,
		ShardsNum: common.DefaultShards, Dim: common.DefaultDim, MaxCapacity: 10}
	collName := createCollection(ctx, t, mc, cp)

	coll, _ := mc.DescribeCollection(ctx, collName)
	for _, field := range coll.Schema.Fields {
		log.Println(fmt.Sprintf("field: %s type is %v, element type is %v", field.Name, field.DataType, field.ElementType))
	}

	// prepare and insert array rows data
	dp := DataParams{CollectionName: collName, PartitionName: "", CollectionFieldsType: AllFields,
		start: 0, nb: common.DefaultNb, dim: common.DefaultDim, EnableDynamicField: false, WithRows: true}
	_, _ = insertData(ctx, t, mc, dp, common.WithArrayCapacity(10))

	// flush and check row count
	errFlush := mc.Flush(ctx, collName, false)
	common.CheckErr(t, errFlush, true)
	stats, _ := mc.GetCollectionStatistics(ctx, collName)
	require.Equal(t, strconv.Itoa(common.DefaultNb), stats[common.RowCount])
}
```

Test logs:
```
2023/11/02 00:21:15 milvus_client.go:14: (ApiRequest): func [DescribeCollection], args: [context.Background.WithDeadline(2023-11-02 00:22:44.541168 +0800 CST m=+120.005295876 [1m29.468314458s]) FdBV]
2023/11/02 00:21:15 milvus_client.go:21: (ApiResponse): func [DescribeCollection], results: [0xc00018ea80]
2023/11/02 00:21:15 insert_test.go:449: field: int64 type is int64, element type is undefined
2023/11/02 00:21:15 insert_test.go:449: field: bool type is bool, element type is undefined
2023/11/02 00:21:15 insert_test.go:449: field: int8 type is int8, element type is undefined
2023/11/02 00:21:15 insert_test.go:449: field: int16 type is int16, element type is undefined
2023/11/02 00:21:15 insert_test.go:449: field: int32 type is int32, element type is undefined
2023/11/02 00:21:15 insert_test.go:449: field: float type is float32, element type is undefined
2023/11/02 00:21:15 insert_test.go:449: field: double type is float64, element type is undefined
2023/11/02 00:21:15 insert_test.go:449: field: varchar type is []byte, element type is undefined
2023/11/02 00:21:15 insert_test.go:449: field: json type is JSON, element type is undefined
2023/11/02 00:21:15 insert_test.go:449: field: floatVec type is []float32, element type is undefined
2023/11/02 00:21:15 insert_test.go:449: field: boolArray type is Array, element type is bool
2023/11/02 00:21:15 insert_test.go:449: field: int8Array type is Array, element type is int8
2023/11/02 00:21:15 insert_test.go:449: field: int16Array type is Array, element type is int16
2023/11/02 00:21:15 insert_test.go:449: field: int32Array type is Array, element type is int32
2023/11/02 00:21:15 insert_test.go:449: field: int64Array type is Array, element type is int64
2023/11/02 00:21:15 insert_test.go:449: field: floatArray type is Array, element type is float32
2023/11/02 00:21:15 insert_test.go:449: field: doubleArray type is Array, element type is float64
2023/11/02 00:21:15 utils.go:834: {[true false true false true false true false true false] [5 6 7 8 9 10 11 12 13 14] [5 6 7 8 9 10 11 12 13 14] [5 6 7 8 9 10 11 12 13 14] [5 6 7 8 9 10 11 12 13 14] [5 6 7 8 9 10 11 12 13 14] [5 6 7 8 9 10 11 12 13 14]}
2023/11/02 00:21:15 milvus_client.go:14: (ApiRequest): func [InsertRows], args: [context.Background.WithDeadline(2023-11-02 00:22:44.541168 +0800 CST m=+120.005295876 [1m29.425153791s]) FdBV  3000]
2023/11/02 00:21:15 milvus_client.go:19: (ApiResponse): func [InsertRows], error: expected unhandled field boolArray
```

**Comment by badgeek** [comment]
Hello, do you have an estimated time for when this issue might be resolved?

**Comment by congqixia** [comment]
@badgeek working on it, plan to release it this week

**Comment by ThreadDao** [comment]
> Hello, do you have an estimated time for when this issue might be resolved?

Fixed~ Try v2.3.3 go-sdk. thx
